README.md

```diff
@@ -93,6 +93,31 @@ For example, on Fedora:
 
 - to add more python dependencies: add to `requirements.txt` and `requirements-workers.txt`
 
+## Database Migrations
+
+Follow steps below for data migrations.
+
+- Checkout the master branch with ensure no schema changes are present in `cachito/web/models.py`
+- Set `SQLALCHEMY_DATABASE_URI` to `sqlite:///cachito-migration.db` in `cachito/web/config.py` under the `Config` class
+- Run `cachito db upgrade` which will create an empty database in the root of your Git repository called `cachito-migration.db` with the current schema applied
+- In case of schema changes,
+  - Apply any schema changes to `cachito/web/models.py`
+  - Run `cachito db migrate` which will autogenerate a migration script in `cachito/web/migrations/versions`
+- In case of no schema changes,
+  - Run `cachito db revision` to create an empty migration script file
+- Rename the migration script so that the suffix has a description of the change
+- Modify the docstring of the migration script
+- Define the schema of any tables you will be modifying. This is so that it captures the schema of the time of the migration and not necessarily what is in models.py since that reflects the latest schema.
```

Comment:
```suggestion
- For data migrations, define the schema of any tables you will be modifying. This is so that it captures the schema of the time of the migration and not necessarily what is in models.py since that reflects the latest schema.
```